conans/client/conan_api.py

```diff
@@ -929,6 +930,29 @@ def get_default_remote(self):
     def get_remote_by_name(self, remote_name):
         return self._cache.registry.remotes.get(remote_name)
 
+    @api_method
+    def get_recipe_revisions(self, reference, remote_name=None):
+        ref = ConanFileReference.loads(str(reference))
+        if not remote_name:
+            return get_local_recipe_revisions(ref, self._cache)
+        else:
+            remote = self.get_remote_by_name(remote_name)
+            if ref.revision:
+                raise ConanException("Cannot list the revisions of a specific revision")
```

Comment:
Is this just to remote-related code? For a local call, can I list the revisions of a specific revision?